Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/battery.yml

```diff
@@ -0,0 +1,52 @@
+#abstract cartridges for battery weapon tricks
+
```

Comment:
Abstract cartridges aren't a thing, it's a waste of entities. The reason it exists is for stuff like CM guns where they have caseless ammo.

Reply:
resolved by the shotgun spread change pr